RELEASE_NOTES

```diff
@@ -1,3 +1,19 @@
+========================================================================
+Amazon CloudWatch Agent 1.247354.0 (2022-07-14)
+========================================================================
+
+Enhancements:
+* Increase PutMetricData request size allowed in agent
+
+========================================================================
+Amazon CloudWatch Agent 1.247353.0 (2022-06-13)
+========================================================================
+
+Enhancements:
+* Upgrade the Telegraf dependency to v1.23.0
+* Drop support for mem_cached metrics on MacOS
+* Fix Windows filepath escaping so paths with spaces work again (#490)
```

Comment:
Nitpick:
This should have been listed as a Bug fix.

Reply:
Ack. Just keeping parity with what was already released. I'm not super partial about differentiating between new features versus bug fixes in the release notes, personally